docs/modules/system preparation.rst

```diff
@@ -0,0 +1,309 @@
+.. _system_preparation:
+
+Preparing systems for constant-pH simulation
```

Comment:
I'll do that. Will open an issue to discuss that strategy. https://github.com/choderalab/constph-openmm/issues/58